src/aiokatcp/server.py

```diff
@@ -475,7 +475,7 @@ def _service_task_done(self, task: asyncio.Task) -> None:
             except AttributeError:
                 # Python 3.7 does not have task names
                 name = "a task"
-            logger.warning("Halting the server because %s raised %s", name, exc)
+            logger.warning("Halting the server because %s raised %s", name, exc, exc_info=True)
```

Comment:
Oops, this is an unrelated change I made to help with debugging service task failures.